modules/users/server/controllers/users/users.authentication.server.controller.js

```diff
@@ -104,7 +104,7 @@ exports.oauthCallback = function (strategy) {
     var sessionRedirectURL = req.session.redirect_to;
     delete req.session.redirect_to;
 
-    passport.authenticate(strategy, function (err, user, redirectURL) {
```

Comment:
The variable name has been changed to make it consistent with Passport.js documentation. Currently its not being used.